examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,218 @@
+               interpolation='nearest')
+    plt.title(title)
+
+
+###############################################################################
+# Load ADHD dataset and MSDL atlas
+# --------------------------------
+
+###############################################################################
+# We study only 20 subjects from the ADHD dataset, to save computation time.
+from nilearn import datasets
+
+adhd_data = datasets.fetch_adhd(n_subjects=20)
+
+###############################################################################
+# We use probabilistic regions of interest (ROIs) from the MSDL atlas.
+msdl_data = datasets.fetch_atlas_msdl()
+msdl_coords = msdl_data.region_coords
+n_regions = len(msdl_coords)
+print('MSDL has {0} ROIs, part of the following networks:\n{1}.'.format(
+    n_regions, msdl_data.networks))
+
+###############################################################################
+# Region signals extraction
+# -------------------------
+
+###############################################################################
+# To extract regions time series, we instantiate a
+# :class:`nilearn.input_data.NiftiMapsMasker` object and pass the atlas the
+# file name to it, as well as filtering band-width and detrending option.
+from nilearn import input_data
+
+masker = input_data.NiftiMapsMasker(
+    msdl_data.maps, resampling_target="maps", t_r=2.5, detrend=True,
+    low_pass=.1, high_pass=.01, memory='nilearn_cache', memory_level=1)
+
+###############################################################################
+# Then we compute region signals and extract useful phenotypic informations.
+adhd_subjects = []
+pooled_subjects = []
+site_names = []
+adhd_labels = []  # 1 if ADHD, 0 if control
+for func_file, confound_file, phenotypic in zip(
+        adhd_data.func, adhd_data.confounds, adhd_data.phenotypic):
+    time_series = masker.fit_transform(func_file, confounds=confound_file)
+    pooled_subjects.append(time_series)
+    is_adhd = phenotypic['adhd']
+    if is_adhd:
+        adhd_subjects.append(time_series)
+
+    site_names.append(phenotypic['site'])
+    adhd_labels.append(is_adhd)
+
+print('Data has {0} ADHD subjects.'.format(len(adhd_subjects)))
+
+###############################################################################
+# ROI-to-ROI correlations of ADHD patients
+# ----------------------------------------
+
+###############################################################################
+# The simpler and most commonly used kind of connectivity is correlation. It
+# models the full (marginal) connectivity between pairwise ROIs. We can
+# estimate it using :class:`nilearn.connectome.ConnectivityMeasure`.
+from nilearn.connectome import ConnectivityMeasure
+
+correlation_measure = ConnectivityMeasure(kind='correlation')
+
+###############################################################################
+# From the list of ROIs time-series for all subjects, the `correlation_measure`
```

Comment:
all subjects -> adhd subjects. Since you are fitting on adhd subjects not on pooled subjects ?